### PR TITLE
completed implementation of createShip()

### PR DESCRIPTION
**Pull request by priyushasunkara**

### CODE REVIEW

File: battleship.py [hunk 1]
```diff
@@ -77,7 +77,15 @@ def emptyGrid(rows, cols):
 Returns: 2D list of ints
 '''
 def createShip():
-    return
+    row_centre=random.randint(1,8)
+    col_centre=random.randint(1,8)
+    orientation=random.randint(0,1)
+    ship_placement=[[],[],[]]
```
Comment (LaxmiNarayanaMurthyVemuri) [inline battleship.py]: @priyushasunkara What does 84th line will do in this function?

**Review by LaxmiNarayanaMurthyVemuri — CHANGES_REQUESTED** [review]

File: battleship.py (hunk 1 above)
Comment (priyushasunkara) [reply battleship.py]: it checks if the orientation of the ship is vertical(0) or horizontal(1)

Comment (LaxmiNarayanaMurthyVemuri) [reply battleship.py]: Thank you for the update in the code. Good job!!!

**Review by LaxmiNarayanaMurthyVemuri — APPROVED** [review]

File: battleship.py (hunk 1 above)
Comment (priyushasunkara) [reply battleship.py]: Sir, Can I merge the code in main?

**Pull request closed without merging**